tests/models/downsample/test_down_sample_meta_data.py

```diff
@@ -0,0 +1,49 @@
+"""Test for the DownSampleMetaData class."""
+
+from cg.apps.housekeeper.hk import HousekeeperAPI
+from cg.models.downsample.downsample_data import DownsampleData
+from cg.store import Store
+from cg.store.models import Family, Sample
+
+
+def test_downsample_meta_data_pass_pre_flight(
+    store_with_case_and_sample_with_reads: Store,
+    downsample_hk_api: HousekeeperAPI,
+    downsample_case_internal_id: str,
+    downsample_sample_internal_id_1: str,
+    number_of_reads_in_millions: int,
+    tmp_path_factory,
+):
+    """Test that the pre-flight checks pass when initialising a DownsampleMetaData class."""
```

Comment:
Not clear to me what pre-flight checks mean in this context

Reply:
renamed